### PR TITLE
CQW-44 open graph tags

### PR DESCRIPTION
**Pull request by AdamMankowski**
## TO DO:
* [X] Add og tags 

---

Review:

* [ ] @kruszczynski
* [ ] @pjar
* [ ] @rpietraszko
* [x] @MagdaMalinowska
* [ ] @aberracja

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -39,6 +39,8 @@
 
 set :images_dir, 'images'
 
+set :root_url, 'http://codequest.com'
```
Comment (pjar) [inline config.rb]: Not the `https` ?

Comment (AdamMankowski) [inline config.rb]: Nope out website is on http and not on https

File: lib/custom_helpers.rb [hunk 2]
```diff
@@ -8,6 +8,12 @@ def description_helper
     data.page.description ? data.page.description : ''
   end
 
+  def facebook_meta_data_url
+    path = request['path'].split('/')
```
Comment (pjar) [inline lib/custom_helpers.rb]: Maybe `request['path'].split('/').first` so you could get rid the line below.

Comment (AdamMankowski) [inline lib/custom_helpers.rb]: `first` returns the first object of a collection(or so it says on internet) - and its not only the first item but sometimes 2 objects

File: source/layouts/website.html.slim [hunk 3]
```diff
@@ -4,6 +4,12 @@ html
     meta charset="utf-8"
     title= title_helper
 
+    meta content="Code quest is a Ruby on Rails and iOS development studio." property="og:title"
+    meta content="website" property="og:type"
+    meta content="1028261210595998" property="fb:app_id"
+    meta content="Create great software with us" property="og:description"
+    meta content="http://codequest.com/images/common/logo-fb.png" property="og:image"
+    meta content="#{facebook_meta_data_url}" property="og:url"
```
Comment (pjar) [inline source/layouts/website.html.slim]: Move to partial please and reuse

Comment (MagdaMalinowska) [inline source/layouts/website.html.slim]: Agree with @pjar 👍

**Pull request merged**